### DISCUSSION TITLE
EHCI Traffic Handling request

### DISCUSSION THREAD

**Discussion by VoltexFPS** [DavidS95/Smokeless_UMAF#78] [General]
This is from scewin:
It makes mouse more responsive also i am not sure how accurate is Help String, and i didn't manage to find it anywhere 

Setup Question	= EHCI Traffic Handling
Help String	= Cause CPU To Break Out From C State When USB EHCI Has Pending Traffic
Token	=2CE	// Do NOT change this line
Offset	=8E
Width	=01
BIOS Default	=[00]Disabled 
Options	=[00]Disabled	// Move "*" to the desired Option
         *[01]Enabled

**Comment by Espionage724** [comment]
I'd disable CPU C-states altogether (not sure what would cause pending traffic on EHCI vs XHCI but sounds like it wouldn't matter if C-States aren't enabled)

But it sounds like it would be better for performance Enabled, while disabled might be better power-efficency. I'm not sure the BIOS should handle that though or how that'd interact with something like irqbalance Linux (what if irqbalance balances EHCI traffic across all cores then causes BIOS to wake em all?); I'd leave it default Disabled.
